src/app/ProtectedRoute/ui/ProtectedRoute.tsx

```diff
@@ -0,0 +1,13 @@
+import { Navigate } from 'react-router';
+
+import { ROUTES } from '~/shared/config/routes';
+
+export const ProtectedRoute = ({ children }: { children: React.ReactNode }) => {
+    const isAuthenticated = localStorage.getItem('token') !== null;
+
+    if (!isAuthenticated) {
+        return <Navigate to={`/${ROUTES.SIGN_IN}`} replace />;
+    }
+
+    return <>{children}</>;
```

Comment:
```suggestion
    return <Layout>{children}</Layout>;
```

и не нужно будет постоянно в него оборачивать